main.js

```diff
@@ -0,0 +1,16 @@
+function daysInYear(year) {
+    try {
+        if (!Number.isInteger(year)) {
+            throw 'not valid';

```

Comment:
Will be better to throw an instance of the Error to have the same contract as default js errors